ocrd_anybaseocr/cli/ocrd_anybaseocr_dewarp.py

```diff
@@ -24,10 +24,10 @@
 from ocrd.decorators import ocrd_cli_options, ocrd_cli_wrap_processor
 
 from ..constants import OCRD_TOOL
-from ..pix2pixhd.options.test_options import TestOptions
-from ..pix2pixhd.models.models import create_model
-from ..pix2pixhd.data.base_dataset import BaseDataset, get_params, get_transform
-from ..pix2pixhd.util.util import tensor2im
+from pix2pixhd.options.test_options import TestOptions
+from pix2pixhd.models.models import create_model
+from pix2pixhd.data.base_dataset import BaseDataset, get_params, get_transform
+from pix2pixhd.util.util import tensor2im
```

Comment:
sry, forgot to update these – strange I did not notice via tests (I guess it must have been an old copy in my venv).

Reply:
CI shows the failure ...